src/components/WizardInline/WizardHeader/WizardHeader.jsx

```diff
@@ -43,12 +32,15 @@ class WizardHeader extends Component {
   state = {};
 
   render = () => {
-    const { currentItemId, setItem, items, showLabels, stepWidth, ...others } = this.props;
+    const { currentItemId, setItem, items, showLabels, stepWidth, title, ...others } = this.props;
 
     return (
       <Fragment>
-        <StyledHero {...others} />
-        <StyledProgressIndicator
+        <h2 className="wizard-inline-header" {...others}>
```

Comment:
Why `{...others}` on this?

Reply:
Not sure, I most likely just reused the StyledHero tag and didn't remove this. Taking off.